src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+
+        System.out.println("Certain machine started its work.");
+
+    }
+
+    @Override
+    public void stopWork() {
+
+        System.out.println("Certain machine stopped working.");

```

Comment:
The message 'Certain machine stopped working.' is too generic. Consider changing it to 'Excavator stopped working.' to specify which machine is stopping.